docs/usage/local.md

```diff
@@ -1,58 +1,76 @@
 # Local testnet
 
-To quickly test and run Lodestar we recommend to start a local testnet. We recommend a simple configuration of two beacon nodes with multiple validators
+To quickly test and run Lodestar we recommend starting a local testnet. We recommend a simple configuration of two beacon nodes with multiple validators
 
 **Terminal 1**
 
-Run a beacon node with 8 validators and default settings. State will be written to .tmp/state.ssz
+Run a beacon node, with 8 validators with the following command. 
 
 ```bash
-./lodestar dev --genesisValidators 8 --reset
+./lodestar dev --genesisValidators 8 --genesisTime 1578787200 --network prater --rootDir </path/to/node1> --reset
```

Comment:
maybe remove `--network prater` here and below

I think this is a little confusing as an example in the docs, folks may think they are actually connecting to the prater testnet

Reply:
may be `network` option can be removed from the dev cli options, so that it will throw error if someone even tries to specify

Reply:
> may be `network` option can be removed from the dev cli options, so that it will throw error if someone even tries to specify

Good tip. I'll see to that

Reply:
@g11tech so I looked into your suggestion. The `network` option is a global option and hence the changes that will be required to make it not applicable when the dev cli option is used feels a bit out of scope to this PR. So I created a separate issue: #3695 for the suggestion to be implemented separately.